klock/src/commonTest/kotlin/com/soywiz/klock/DateTest.kt

```diff
@@ -39,5 +43,8 @@ class DateTest {
 		}
 
 		assertEquals("Wed, 18 Sep 2019 00:00:00 UTC", (Date(2019, Month.September, 17) + Time(24)).toString())
+        assertEquals("Tue, 17 Sep 2019 00:00:00 UTC", (Date(2019, Month.September, 18) - Time(24)).toString())
```

Comment:
I know that with the plus the tests were a bit lacky, but can you add a few more tests including minutes, seconds / milliseconds, just to be sure it is working properly?

Reply:
Done